pyccel/codegen/python_wrapper.py

```diff
@@ -173,8 +175,10 @@ def create_shared_library(codegen,
     codegen.ast.set_name(sharedlib_modname)
     wrapper_codegen = CWrapperCodePrinter(codegen.parser.filename, language)
     Scope.name_clash_checker = name_clash_checkers['c']
-    wrapper = CToPythonWrapper(base_dirpath)
-
+    if language == 'cuda':
+        wrapper = CudaToCWrapper(base_dirpath)
+    else:
+        wrapper = CToPythonWrapper(base_dirpath)
```

Comment:
This doesn't look right to me. Shouldn't `CudaToCWrapper` go in a similar place to `FortranToCWrapper` ? If you do this here then won't you lose the Python interface?

Reply:
fixed bafb766